test.ini

```diff
@@ -9,7 +9,7 @@ host = 0.0.0.0
 port = 5000
 
 [app:main]
-use = config:../../src/ckan/test-core.ini
+use = config:../ckan/test-core.ini
```

Comment:
No previous configuration of the validation tests has taken place, this is a required change to run any tests for validation at all.